src/index.html

```diff
@@ -7,12 +7,57 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
     <title>Stars</title>
-    <link
-      rel="stylesheet"
-      href="./style.css"
-    />
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+
+    <div class="containerpai">
+    <div class="containerfilho">
+      <div class="linha1">
```

Comment:
And on all of these following.
Also, please do not mix languages in your code. Naming some classes in English and others in Portuguese isn't a good practice.